common/web/types/src/deps/xml2js/parser.js

```diff
@@ -323,7 +323,11 @@ export class Parser extends EventEmitter {
       } catch (error1) {
         err = error1;
         if (!(this.saxParser.errThrown || this.saxParser.ended)) {
-          this.emit('error', err);
+          if(this.listenerCount('error') > 0) {
+            this.emit('error', err);
+          } else {
+            throw err;
+          }
```

Comment:
For other reviewers, just in case:

We use `eventemitter3` so that Web can safely compile against common/web/types; `events` is a Node-specific package that won't exist in the browser.  There is _one_ particular point of divergence between the two types:

- In `events`, `.emit('error')` will actually _`throw`_ the error if there are no registered listeners.
- In `eventemitter3`, this specialized behavior is omitted.  It will never `throw` on `.emit('error')` regardless of listener count.

This is the one spot in `xml2js` that uses `.emit('error')`, so we opted for a precise adjustment to accommodate for the difference between the two implementations that will replicate the behavior of the original.  One of the `common/web/types` unit tests fails otherwise due to reliance on the pattern.